content/docs/guides/go-application.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Instrumenting a Go application
+---
+
+# Instrumenting a Go application for Prometheus
+
+Prometheus has an official [Go client library](https://github.com/prometheus/client_golang) that you can use to instrument Go applications. In this guide, we'll create a simple Go application that exposes Prometheus metrics via HTTP.
+
+NOTE: For comprehensive API documentation, see the [GoDoc](https://godoc.org/github.com/prometheus/client_golang) for Prometheus' various Go libraries.
+
+## Installation
+
+You can install the `prometheus`, `promauto`, and `promhttp` libraries necessary for the guide using [`go get`](https://golang.org/doc/articles/go_command.html):
+
+```bash
+go get github.com/prometheus/client_golang/prometheus
```

Comment:
You're not using this one anymore

Reply:
It's still (again?) used in the custom metrics example.

Reply:
Ah, missed that